src/oas2/__tests__/accessors.test.ts

```diff
@@ -216,11 +216,6 @@ describe('accessors', () => {
   });
 
   describe('getProduces', () => {
-    test('given all empty arrays should return asterisk', () => {
-      expect(getProduces({}, {})).toEqual(['*/*']);
```

Comment:
would
```
expect(getProduces({}, {})).toEqual([])
```
be OK?

Reply:
@lag-of-death yes, that should be what the test is. The same thing for consumes as well.

@XVincentX What happens if there is a global consumes/produces with json in it, and then an operation has an empty array for consumes/produces?

Reply:
@pytlesk4 Basically the local overrides the global, so you would still get the local content.

Reply:
```suggestion
    test('given all empty arrays should return an empty array', () => {
      expect(getProduces({}, {})).toEqual([]);
      expect(getProduces({ produces: [] }, { produces: [] })).toEqual([]);
    });
```

Just add this test back, but test for an empty array.